ci-operator/jobs/infra-image-mirroring.yaml

```
@@ -239,6 +239,55 @@ periodics:
         - name: config
           configMap:
             name: image-mirror-mappings
+  - agent: kubernetes
+    labels:
+      ci.openshift.io/role: image-mirroring
+      ci.openshift.io/area: codeready-toolchain
+    cron: "@midnight"
+    decorate: true
+    name: periodic-image-mirroring-codeready-toolchain
+    skip_cloning: true
+    spec:
+      containers:
+        - args:
+            - -c
+            - |
+              set -o errexit
+              failures=0
+              for mapping in /etc/imagemirror/mapping_codeready_toolchain*; do
+                echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes"
+                if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
+                  echo "ERROR: Failed to mirror images from $mapping"
+                  failures=$((failures+1))
+                fi
+              done
+              exit $failures
+          command:
+            - /bin/bash
+          env:
+            - name: HOME
+              value: /home/mirror
+          image: "quay.io/openshift/origin-cli:4.1"
+          imagePullPolicy: Always
+          name: ""
+          resources:
+            requests:
+              cpu: 500m
+          volumeMounts:
+            - mountPath: /home/mirror/.docker/config.json
+              name: push
+              readOnly: true
+              subPath: .dockerconfigjson
```

```suggestion
              subPath: config.json
```
Sorry about this bug in the template, I'll fix it